libraries/Filter/HarmonicNotchFilter.cpp

```diff
@@ -360,6 +422,82 @@ void HarmonicNotchFilter<T>::reset()
     }
 }
 
+#if HAL_LOGGING_ENABLED
+// @LoggerMessage: FCN
+// @Description: Filter Center Message - per motor
+// @Field: TimeUS: microseconds since system startup
+// @Field: I: instance
+// @Field: NDn: number of active dynamic harmonic notches
+// @Field: CF1: centre frequency for motor 1
+// @Field: CF2: centre frequency for motor 2
+// @Field: CF3: centre frequency for motor 3
+// @Field: CF4: centre frequency for motor 4
+// @Field: CF5: centre frequency for motor 5
+// @Field: HF1: 2nd harmonic frequency for motor 1
```

Comment:
"Second harmonic centre frequency for motor 1"

Reply:
changed